<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graphics_ProgrammingProject1Class</class>
 <widget class="QMainWindow" name="Graphics_ProgrammingProject1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bar Charts Generator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="optionsLayout">
      <item>
       <layout class="QVBoxLayout" name="valuesLayout">
        <item>
         <widget class="QLabel" name="chartValuesLabel">
          <property name="text">
           <string>Chart Values</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="barAValueLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Bar A</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="valueASpinBox">
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="barBValueLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Bar B</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="valueBSpinBox">
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="colorsLayout">
        <item>
         <widget class="QLabel" name="chartColorsLabel">
          <property name="text">
           <string>Chart Colors</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="barAColorLayout">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Bar A</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="colorABtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="barBColorLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Bar B</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="colorBBtn">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="resolutionLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Output Resolution</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="resolutionCombo">
            <item>
             <property name="text">
              <string>640 x 480</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>800 x 600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1024 x 760</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="saveBtn">
        <property name="text">
         <string>Save Chart</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>863</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Graphics_ProgrammingProject1.qrc"/>
 </resources>
 <connections/>
</ui>
